pallets/automation-time/src/weights.rs

```diff
@@ -83,62 +85,76 @@ impl<T: frame_system::Config> WeightInfo for AutomationWeight<T> {
+	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
 	// Storage: AutomationTime ScheduledTasks (r:1 w:1)
 	fn cancel_scheduled_task() -> Weight {
-		(14_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+		(18_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	// Storage: AutomationTime Tasks (r:1 w:1)
-	// Storage: AutomationTime ScheduledTasks (r:1 w:1)
+	// Storage: Timestamp Now (r:1 w:0)
+	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
+	// Storage: AutomationTime ScheduledTasks (r:24 w:24)
 	fn cancel_scheduled_task_full() -> Weight {
-		(23_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(2 as Weight))
-			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+		(100_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(27 as Weight))
+			.saturating_add(T::DbWeight::get().writes(25 as Weight))
 	}
 	// Storage: AutomationTime Tasks (r:1 w:1)
+	// Storage: Timestamp Now (r:1 w:0)
+	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
 	// Storage: AutomationTime ScheduledTasks (r:1 w:0)
-	// Storage: AutomationTime TaskQueue (r:1 w:1)
 	fn cancel_overflow_task() -> Weight {
 		(23_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(3 as Weight))
-			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+			.saturating_add(T::DbWeight::get().reads(4 as Weight))
+			.saturating_add(T::DbWeight::get().writes(1 as Weight))
 	}
 	// Storage: AutomationTime Tasks (r:1 w:1)
+	// Storage: Timestamp Now (r:1 w:0)
+	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
 	// Storage: AutomationTime ScheduledTasks (r:1 w:1)
 	fn force_cancel_scheduled_task() -> Weight {
-		(14_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+		(18_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	// Storage: AutomationTime Tasks (r:1 w:1)
+	// Storage: Timestamp Now (r:1 w:0)
+	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
 	// Storage: AutomationTime ScheduledTasks (r:1 w:1)
 	fn force_cancel_scheduled_task_full() -> Weight {
-		(23_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(2 as Weight))
+		(38_000_000 as Weight)
+			.saturating_add(T::DbWeight::get().reads(4 as Weight))
 			.saturating_add(T::DbWeight::get().writes(2 as Weight))
 	}
 	// Storage: AutomationTime Tasks (r:1 w:1)
-	// Storage: AutomationTime ScheduledTasks (r:1 w:0)
-	// Storage: AutomationTime TaskQueue (r:1 w:1)
-	fn force_cancel_overflow_task() -> Weight {
-		(23_000_000 as Weight)
-			.saturating_add(T::DbWeight::get().reads(3 as Weight))
-			.saturating_add(T::DbWeight::get().writes(2 as Weight))
+	// Storage: Timestamp Now (r:1 w:0)
+	// Storage: AutomationTime LastTimeSlot (r:1 w:0)
+	// Storage: AutomationTime ScheduledTasks (r:24 w:0)
+	fn force_cancel_overflow_task_full() -> Weight {
```

Comment:
because this test never scheduled the task just put one straight on the task queue.  so it is looking for all executions times and not finding any.  https://github.com/OAK-Foundation/OAK-blockchain/pull/128/files/2d244949e795dd25c452201e06a24a89e1ed9474#diff-eb0580064fc58a5d272e21ab88621358160d3d67ea4a20f6fea4161373e5e15fR198

Reply:
fixed.